pysatNASA/instruments/methods/cdaweb.py

```diff
@@ -113,7 +192,179 @@ def load(fnames, tag=None, inst_id=None, file_cadence=dt.timedelta(days=1),
+        # metadata for pysat using some assumptions. Depending upon your needs
+        # the resulting pandas DataFrame may need modification
+        ldata = []
+        for lfname in fnames:
+            temp_data = cdflib.cdf_to_xarray(lfname, to_datetime=True)
+            ldata.append(temp_data)
+        # Combine individual files together
+        if len(ldata) > 0:
+            data = xr.combine_by_coords(ldata)
+
+    all_vars = io.xarray_all_vars(data)
+
+    # Convert output epoch name to 'time' for pysat consistency
+    if epoch_name != 'time':
+        if 'time' not in all_vars:
+            if epoch_name in data.dims:
+                data = data.rename({epoch_name: 'time'})
+            elif epoch_name in all_vars:
+                data = data.rename({epoch_name: 'time'})
+                wstr = ''.join(['Epoch label: "', epoch_name, '"',
+                                ' is not a dimension.'])
+                pysat.logger.warning(wstr)
+            else:
+                estr = ''.join(['Epoch label: "', epoch_name, '"',
+                                ' not found in loaded data, ',
+                                repr(all_vars)])
+                raise ValueError(estr)
+
+        epoch_name = 'time'
+
+    all_vars = io.xarray_all_vars(data)
+
+    meta = pysat.Meta(labels=labels)
+
+    full_mdict = {}
+
+    if meta_translation is None:
+        # Assign default translation using `meta`
+        meta_translation = {}
+
+    # Drop metadata labels initialization
+    if drop_meta_labels is None:
+        drop_meta_labels = []
+    else:
+        drop_meta_labels = pysat.utils.listify(drop_meta_labels)
+
+    for key in all_vars:
+        meta_dict = {}
+        for nc_key in data[key].attrs.keys():
+            meta_dict[nc_key] = data[key].attrs[nc_key]
+        full_mdict[key] = meta_dict
+        data[key].attrs = {}
+
+    for data_attr in data.attrs.keys():
+        setattr(meta.header, data_attr, getattr(data, data_attr))
+
+    # Process the metadata. First, drop labels as requested.
+    for var in full_mdict:
+        for label in drop_meta_labels:
+            if label in full_mdict[var]:
+                full_mdict[var].pop(label)
+
+    # Second, remove some items pysat added for netcdf compatibility.
+    filt_mdict = io.remove_netcdf4_standards_from_meta(full_mdict, epoch_name,
+                                                       meta.labels)
+
+    # Translate labels from file to pysat compatible labels using
+    # `meta_translation`
+    filt_mdict = io.apply_table_translation_from_file(meta_translation,
+                                                      filt_mdict)
+
+    # Next, allow processing by developers so they can deal with
+    # issues with specific files
+    if meta_processor is not None:
+        filt_mdict = meta_processor(filt_mdict)
+
+    # Meta cannot take array data, if present save it as seperate meta data
+    # labels.
+    filt_mdict = io.meta_array_expander(filt_mdict)
```

Comment:
I double checked the order of the meta processing functions against the order in pysat. Checks out 👍